inbox/encryption/vault.py

```diff
@@ -0,0 +1,76 @@
+import base64
+import os
+import time
+
+import hvac
+
+from inbox.config import config
+
+vault_config = config.get('VAULT')
+
+
+client = hvac.Client(url=vault_config['URL'])
+
+client.auth_approle(vault_config['APP_ROLE_ID'], vault_config['APP_ROLE_SECRET_ID'])
+
+
+def encrypt(plaintext, namespace_public_id):
+    start = time.time()
+
+    if not plaintext:
+        return ''
+
+    if not namespace_public_id:
+        return ''
+
+    # TODO: add proper handling of exception
+    try:
+        result = client.write(
+            # 'transit/encrypt/' + namespace_public_id,
```

Comment:
It looks like mistake from debugging. Isn't it?

Reply:
Sorry, indeed.